src/main/java/org/plumelib/util/StringsPlume.java

```diff
@@ -308,6 +308,7 @@ public static String escapeJava(String orig) {
       char c = orig.charAt(i);
       switch (c) {
         case '\"':
+        case '\'':
```

Comment:
This isn't needed for strings.

Also, please add a test when you change behavior.

Reply:
Sorry about that, it was very stupid. I don't know what i was thinking.